### DISCUSSION TITLE
How to copy the URL of a current tab and how to pagedown on Vomnibar using keyboard?

### DISCUSSION THREAD

**Discussion by Samueru-sama** [gdh1995/vimium-c#1213] [Q&A] [answered]
I've been using vimium-c with the zen-browser.

* The first issue is because on firefox based browsers, if you do ctrl+l to focus on the url bar, pressing escape does not remove the focus and one has to use F6 instead to go back to the page (I hate it). So is there a way that I could copy the url of the currently active tab using vimium-c? since that's usually why I need to focus on the url bar.

* The other issue is that when a have many tabs open, moving down the vomnibar using the arrow keys does not change to the next page, I instead need to use the mousewheel for that, is it possible to use the keyboard instead?

**Comment by gdh1995** [comment, marked as answer]
You may try `copyCurrentUrl`, which should work in most common situations and has been mapped to `yy` by default.

You may also use `PageDown`, `PageUp`, `Shift+ArrowDown` and `Shift+ArrowUp` to switch to next 11-20, 21-30, ... suggestions.

**Reply by Samueru-sama** [reply]
Thank you, I read the available keybinds page many times and `yy` escaped me because the description says "Copy page's info" instead of url/link. 

>You may also use PageDown, PageUp, Shift+ArrowDown and Shift+ArrowUp to switch to next 11-20, 21-30, ... suggestions.

I noticed an issue though, if I only have say 3 tabs open I can still change pages and that just causes the same tabs to be repeated over and over, so it is not clear when I have more tabs or if it is the same tab I saw before.
